LICENSE

```diff
@@ -1,4 +1,5 @@
-Copyright (c) 2016, Vighnesh Birodkar and scikit-learn-contrib contributors
```

Comment:
Une petite mention à QM et Michelin ?

Reply:
Dans le readme oui. dans la license, l'usage est plutôt : "and all contributors".